cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -1456,7 +1456,7 @@ func (s *initramfsMountsSuite) testInitramfsMountsInstallRecoverModeStep2Measure
 }
 
 func (s *initramfsMountsSuite) TestInitramfsMountsInstallModeStep2Measure(c *C) {
-	s.testInitramfsMountsInstallRecoverModeStep2Measure(c, "")
+	s.testInitramfsMountsInstallRecoverModeStep2Measure(c, "install")
```

Comment:
We probably want to test both empty and install. Back in the day the rational was that if you don't have any grubenv (fresh install) we assumed this is install mode. I think we either need to test that we still interpret things this way or drop it.